### PR TITLE
Ek

### PR DESCRIPTION
**Pull request by ekuiter**
updated my SPLC papers and added some other papers

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -3277,6 +3282,109 @@ @inproceedings{SL:VaMoS21
...
+}
+
+@inproceedings{BRS:GPCE20,
+	author    = {Tabea Bordis and Tobias Runge and Ina Schaefer},
+	title     = {{Correctness-by-Construction for Feature-Oriented Software Product Lines}},
+	booktitle = GPCE,
+	publisher = ACM,
+	pages     = {22--34},
+	doi       = {10.1145/3425898.3426959},
+	year      = 2020
```
Comment (pmbittner) [inline literature.bib:3311]: address missing

File: literature.bib [hunk 2]
```diff
@@ -3277,6 +3282,109 @@ @inproceedings{SL:VaMoS21
...
+}
+
+@inproceedings{S:APLAS20,
+	author    = {Steinh\"{o}fel, Dominic},
+	title     = {{REFINITY to Model and Prove Program Transformation Rules}},
+	booktitle = APLAS,
+	publisher = Springer,
+	pages     = {311--319},
+	doi       = {10.1007/978-3-030-64437-6\_16},
```
Comment (pmbittner) [inline literature.bib:3384]: Is it necessary to escape the backspace here? (just for curiosity)

File: literature.bib [hunk 3]
```diff
@@ -3277,6 +3282,109 @@ @inproceedings{SL:VaMoS21
...
+}
+
+@inproceedings{KMB:SPLC20,
+	author    = {Kr{\"u}ger, Jacob and Mahmood, Wardah and Berger, Thorsten},
+	title     = {{Promote-pl: A Round-Trip Engineering Process Model for Adopting and Evolving Product Lines}},
+	booktitle = SPLC,
+	publisher = ACM,
+	pages     = {1--12},
+	doi       = {10.1145/3382025.3414970},
+	year      = 2020
```
Comment (pmbittner) [inline literature.bib:3375]: address missing

File: literature.bib [hunk 4]
```diff
@@ -3277,6 +3282,109 @@ @inproceedings{SL:VaMoS21
...
+}
+
+@inproceedings{GBKR:SPLC20,
+	author    = {Gr{\"u}ner, Sten and Burger, Andreas and Kantonen, Tuomas and R{\"u}ckert, Julius},
+	title     = {{Incremental Migration to Software Product Line Engineering}},
+	booktitle = SPLC,
+	publisher = ACM,
+	pages     = {1--11},
+	doi       = {10.1145/3382025.3414956},
+	year      = 2020
```
Comment (pmbittner) [inline literature.bib:3343]: address missing

**Review by pmbittner — COMMENTED** [review]
Hi Elias,

thanks for updating some publications for #110 . The changes look good, there is only a few addresses missing.

Also, I created the `Dommeldange` variable on main (sorry, I saw your PR too late and pushed already), which is the location for this year's SPLC. Could you rebase / merge main and then use this variable for the `location` of the new SPLC publications? Afterwards, we can merge.

File: literature.bib (hunk 2 above)
Comment (ekuiter) [reply literature.bib:3384]: good catch. apparently not, judging from the rest of the file

**Comment by ekuiter** [comment]
thanks for reviewing! I updated the entries of the SPLC, maybe you can have a look again?

**Review by pmbittner — APPROVED** [review]

**Pull request merged**